p/.htaccess

```diff
@@ -27,6 +27,9 @@ AddDefaultCharset	UTF-8
 	<FilesMatch "\.php$">
 		ExpiresActive	Off
 	</FilesMatch>
+	<FilesMatch "oidc">
```

Comment:
Could you try to make a slightly more efficient match, for instance on the directory `/i/oidc/` or using a regex anchor `^` such as `^/i/oidc/` ?
Could also be wrapped into a `<IfDefine OIDC_ENABLED>`

Reply:
Simplified in https://github.com/FreshRSS/FreshRSS/pull/5351/commits/c29ed85a8531b48e7f6615fe3be43b8faab8727c
Double-check welcome